app/controllers/application_controller.rb

```diff
@@ -18,4 +19,9 @@ def authenticate_user!
       redirect_to new_user_session_path, :notice => 'please sign in'
     end
   end
+
+def get_emergencies_list
+    @emergencies_list = Emergency.last(5).reverse
```

Comment:
>  It should be by updated_at
ups... I want use this - Emergency.all.order('updated_at').last(5), but I tested inquiries to base, and forgot change request

> and why it's in before_action?
I know, but without it, I have error: undefined method `each' for nil:NilClass in app/views/visitors/index.html.slim